package.json

```diff
@@ -52,10 +53,14 @@
     "postcss-html": "^1.5.0",
     "squire-rte": "1.11.3",
     "sweetalert2": "11.4.33",
+    "tsc-watch": "^5.0.3",
     "zxcvbn": "4.4.2"
   },
   "scripts": {
     "build": "./build.sh",
+    "sync": "./sync.sh",
+    "watch": "tsc-watch -p ./tsconfig.json --onSuccess 'npm run sync'",
+    "start": "concurrently \"npm run watch\" \"npm run run_chromium\"",
```

Comment:
@tomholub I think we'd better stick with current implementation.
tsc-watch only watches ts files and doesn't watch changes for .html, manifest.json, etc.
More over when we run `watch` with `run_chromium`, chromium active page(s) gets removed when user changes something as we remove whole build directory and re-build it again.

With current build mechanism, it seems hard/impossible to implement hot-reload.

Reply:
Let me know what you think and what you have in mind.

Reply:
ok